modular_zubbers/code/modules/ashwalker_names/ashwalker_names.dm

```diff
@@ -0,0 +1,80 @@
+/proc/generate_one_word_objects()
+
+	. = list()
+
+	var/regex/regex_inclusion = regex(@'^(\w+)$',"i") //limits to things with one word
+
+	for(var/obj/object as anything in subtypesof(/obj))
+		var/object_name = initial(object.name)
+		if(!object_name || !regex_inclusion.Find(object_name))
+			continue
+		.[capitalize(lowertext(object_name))] = TRUE
```

Comment:
Someone's going to look at this in the future and go "wtf"

Reply:
The assoc list is so duplicate entries don't get added.